### DISCUSSION TITLE
Does it run scripts on Winows Server?

### DISCUSSION THREAD

**Discussion by capsloq** [adnanh/webhook#523] [Q&A] [answered]
Hello,

i was wondering if the lib does run on Windows Server, and if so, is it possible to execute .bat or .ps (powershell) scripts?
I only saw examples for linux execute commands like /home/user/script.sh

Thank you.

**Comment by moorereason** [comment, marked as answer]
Yes.  For a PowerShell example, see https://github.com/adnanh/webhook/issues/483#issuecomment-733775846.

We should add this to the docs.

**Reply by capsloq** [reply]
Thank you, solved
